libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -269,6 +269,9 @@ void GCS_MAVLINK::send_battery_status(const uint8_t instance) const
     } else {
         consumed_wh = -1;
     }
+    uint8_t percentage = -1;
+    IGNORE_RETURN(battery.capacity_remaining_pct(percentage, instance));
```

Comment:
I think this works, but we should probably have done something like this:
```suggestion
    uint8_t _percentage = 0;
    const int8_t percentage = battery.capacity_remaining_pct(percentage, instance) ? percentage : -1;
```

Reply:
Yes, that looks good. I was trying to avoid creating another variable. but doing uint = -1 didn't feel good anyway.

This solution looks clean, thanks!